main.go

```diff
@@ -44,6 +44,7 @@ func main() {
 	kingpin.Flag("server-insecure-tls", "Skip PMM Server TLS certificate validation").BoolVar(&commands.GlobalFlags.ServerInsecureTLS)
 	kingpin.Flag("debug", "Enable debug logging").BoolVar(&commands.GlobalFlags.Debug)
 	kingpin.Flag("trace", "Enable trace logging (implies debug)").BoolVar(&commands.GlobalFlags.Trace)
+	kingpin.Flag("pmm-agent-listen-port", "Overwrite default listen port of pmm-agent").Default("7777").Uint32Var(&commands.GlobalFlags.PMMAgentListenPort)
```

Comment:
How about _Set listen port of pmm-agent_ instead ?

Reply:
@ShashankSinha252 Listen port is already used as a flag for add *. I cannot use same name for global flag. Thats why I am using whole name pmm-agent-listen-port to make it more clear even in code. Do you still persist on change name of param pmmAgentListenPort to listenPort in SetTransport func which you commented above?

Reply:
Ah, I should have been more specific. I only meant to suggest `Overwrite default listen port of pmm-agent` -> `Set listen port of pmm-agent`.

Reply:
Ah I see. Yes sure agree.